webindex/modules/data/src/test/java/webindex/data/fluo/PageLoaderTest.java

```diff
@@ -0,0 +1,63 @@
+package webindex.data.fluo;
+
+import java.net.MalformedURLException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import webindex.core.models.Page;
+import webindex.core.models.URL;
+
+@SuppressWarnings("unused")
+public class PageLoaderTest {
+
+	@Test
+	public void testUpdatePageWithNullPage() {
+		Page p = null;
+		PageLoader loader = PageLoader.updatePage(p);
+		loader = null;
+	}
+	
+	@Test(expected = IllegalArgumentException.class)
+	public void testEmptyPageThrowsIllegalArgument() {
+		Page p = Page.EMPTY;
+		PageLoader loader = PageLoader.updatePage(p);
+		loader = null;
+	}
+	
+	@Test(expected = NullPointerException.class)
+	public void testDeleteFailsWithNullUrl() {
+		try {
+			PageLoader loader = PageLoader.deletePage(null);
+		} catch (MalformedURLException e) {
+			e.printStackTrace();
+		}
+	}
+	
+	@Test
+	public void testDeletePage() {
+		URL url = new URL("","","",0,false,false);
+		try {
+			PageLoader loader = PageLoader.deletePage(url);
+		} catch (MalformedURLException e) {
+			e.printStackTrace();
+		}
+	}
+	
+	@Test
```

Comment:
Should this be `@Test(expected = NullPointerException.class)`?  Also if that change is made seem like all of the catch blocks could be dropped.

Reply:
@kpm1985 do you know if this is still an issue?